orttraining/orttraining/core/optimizer/bias_dropout_fusion.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/optimizer/graph_transformer.h"
+
+namespace onnxruntime {
+
+/**
+@Class BiasDropoutFusion
+
+Fuse Add + Dropout + optional Add to BiasDropoutFusion
+
+*/
+class BiasDropoutFusion : public GraphTransformer {
```

Comment:
Why don't we make this a RewriteRule? From a quick skim, it seems that the fusion is quite local, so we can avoid traversing the whole tree and call Resolve.